src/crafting_gui.cpp

```diff
@@ -591,100 +624,14 @@ static void draw_recipe_subtabs( WINDOW *w, std::string tab, std::string subtab,
         mvwputch( w, i, width - 1, BORDER_COLOR,  LINE_XOXO ); // ^|
     }
 
-    switch( mode ) {
```

Comment:
Ah, pointless whitespace changes. Please read the [code style policy](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CODE_STYLE.txt). The current spacing here is correct and should be left as is. Beside: your new style is inconsistent anyway: compare `switch (mode) {` here to `switch ( mode ) {` at line 586.